CHANGELOG.md

```diff
@@ -26,7 +26,7 @@ Our versioning strategy is as follows:
 * `[sitecore-jss]` _GraphQLRequestClient_ now can accept custom 'headers' in the constructor or via _createClientFactory_ ([#1806](https://github.com/Sitecore/jss/pull/1806))
 * `[templates/nextjs]` Removed cors header for API endpoints from _lib/next-config/plugins/cors-header_ plugin since cors is handled by API handlers / middlewares ([#1806](https://github.com/Sitecore/jss/pull/1806))
 * `[sitecore-jss-nextjs]` Updates to Next.js editing integration to further support secure hosting scenarios (on XM Cloud & Vercel) ([#1832](https://github.com/Sitecore/jss/pull/1832))
-* `[templates/nextjs-xmcloud]` `[sitecore-jss]` AB testing and componente level personalization support. ([#1844](https://github.com/Sitecore/jss/pull/1844))
+* `[templates/nextjs-xmcloud]` `[sitecore-jss]` A/B testing and component level personalization support. ([#1844](https://github.com/Sitecore/jss/pull/1844)) ([#1848](https://github.com/Sitecore/jss/pull/1848))
```

Comment:
I think we still need to add a brief list of breaking changes here, like we do under "breaking change" section

Reply:
Added breaking changes for `CdpHelper`